api/db-cdr/changelog-schema/db.changelog-24.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="srushtigangireddy" id="changelog-24">
+    <validCheckSum>ANY</validCheckSum>
+    <addColumn tableName="survey_module">
+      <!-- To display the surveys in the release order -->
```

Comment:
Is this related to this PR?

Reply:
This pr addresses DB-161, DB-168, DB-159 and DB-169.